packages/create-hint/package.json

```diff
@@ -10,7 +10,11 @@
   "bin": {
     "create-hint": "./dist/src/create-hint.js"
   },
-  "dependencies": {},
+  "dependencies": {
+      "@hint/utils-fs": "^1.0.0",
+      "@hint/utils-string": "^1.0.0",
+      "@hint/utils-types": "^1.0.0"
```

Comment:
Any particular reason these are `dependencies` instead of `devDependencies`? I notice `@hint/utils` and `@hint/utils-tests-helpers` are still listed as the latter.

Reply:
`Category` is an `enum`, so we need it as dependency.

> I notice @hint/utils and @hint/utils-tests-helpers are still listed as the latter

what is the problem with that?

I notice that @hint/utils should be a dependecy, not a devDependency.